### ISSUE TITLE
Specifying alt text for site-wide social metadata image?

### ISSUE THREAD

**Issue by dragonstyle** [quarto-dev/quarto-cli#6732]
### Discussed in https://github.com/quarto-dev/quarto-cli/discussions/6731

<div type='discussions-op-text'>

<sup>Originally posted by **kthayashi** September  5, 2023</sup>
### Description

Is there currently a way to specify alt text for images in site-wide social metadata (`twitter-card`, `open-graph`) for a Quarto website? I've run into an issue where, when an image with alt text is already specified on a page (e.g. on an About page), specifying an `image` in `_quarto.yml` overrides the existing image but fails to alter the alt text in the HTML metadata for that page. Here's some code for a minimal website to reproduce this issue:

`_quarto.yaml`:
```yml
project:
  type: website

website:
  title: Test
  navbar:
    left:
      - href: "index.qmd"
        text: About
  site-url: "https://www.example.com/"
  twitter-card:
    description: Description
    image: "img/thumbnail.png"
    image-alt: Thumbnail
  open-graph:
    description: Description
    image: "img/thumbnail.png"
    image-alt: Thumbnail
```

YAML header for `index.qmd`:
```yaml
title: About
image: "img/headshot.png"
image-alt: Headshot
about:
  template: trestles
```

Social metadata in rendered `_site/index.html`:
```html
<meta property="og:title" content="Test - About">
<meta property="og:description" content="Description">
<meta property="og:image" content="https://www.example.com/img/thumbnail.png">
<meta property="og:site-name" content="Test">
<meta property="og:image:height" content="40">
<meta property="og:image:width" content="40">
<meta property="og:image:alt" content="Headshot">
<meta name="twitter:title" content="Test - About">
<meta name="twitter:description" content="Description">
<meta name="twitter:image" content="https://www.example.com/img/thumbnail.png">
<meta name="twitter:image-height" content="40">
<meta name="twitter:image-width" content="40">
<meta name="twitter:image:alt" content="Headshot">
<meta name="twitter:card" content="summary_large_image">
```

The image pointed to in the HTML metadata does not match the associated alt text, and specifying `image-alt` in `_quarto.yml` does not seem to do anything. Or is specifying a site-wide social metadata image in `_quarto.yml` not an expected use case?</div>
